hw/top_darjeeling/ip_autogen/rstmgr/data/rstmgr.hjson

```diff
@@ -35,10 +35,8 @@
   clocking: [
     {clock: "clk_i", reset: "rst_ni", primary: true},
     {clock: "clk_aon_i"},
-    {clock: "clk_io_div4_i"},
-    {clock: "clk_main_i"},
     {clock: "clk_io_i"},
-    {clock: "clk_io_div2_i"},
+    {clock: "clk_main_i"},
     {clock: "clk_usb_i"},
```

Comment:
This usb clock should be gone when you rebase after https://github.com/lowRISC/opentitan/pull/26382.

Reply:
Aware of it. That's why I did not change it here.